DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -2757,6 +2757,9 @@ private void BracketWhitespace() {
 
                         return new DMASTRgb(identifier.Location, callParameters);
                     }
+                    case "animate": {
```

Comment:
This should verify that there is at least 1 argument.
```suggestion
                    case "animate": {
                        if (callParameters.Length is < 1)
                            Emit(WarningCode.InvalidArgumentCount, callLoc,
                                "Expected at least 1 argument for animate()");
                        
```